core/field_angle.js

```diff
@@ -35,21 +35,17 @@ goog.require('goog.userAgent');
 
 /**
  * Class for an editable angle field.
- * @param {(string|number)=} opt_value The initial content of the field. The
- *     value should cast to a number, and if it does not, '0' will be used.
+ * @param {*} opt_value The initial value of the field. The
+ *     value should cast to a number, and if it does not, 0 will be used.
  * @param {Function=} opt_validator An optional function that is called
  *     to validate any constraints on what the user entered.  Takes the new
- *     text as an argument and returns the accepted text or null to abort
+ *     angle as an argument and returns the accepted angle or null to abort
  *     the change.
  * @extends {Blockly.FieldTextInput}
  * @constructor
  */
 Blockly.FieldAngle = function(opt_value, opt_validator) {
-  // Add degree symbol: '360°' (LTR) or '°360' (RTL)
```

Comment:
Making sure I understand:
Removing the degree symbol stuff here isn't specifically because of the change to the validator--it's because it was previously dynamic (depended on RTL) and is now always on the same side of the number, so it can just be an additional textNode that we don't change.  

Is that right?

Reply:
Correct! I had to switch up the rendering because of how render_ changed to handle htmlInput being red or not.

Reply:
Note to self: I still need to dig into the rendering change on textinput, and how it reflects back into other fields.